### PR TITLE
Add js

### PR DESCRIPTION
**Pull request by celinevo**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,111 +9,271 @@
...
+            <li>Tag 1</li>
+            <li>Tag 2</li>
+            <li>Tag 3</li>
+            <li>Tag 4</li>
+          </ul>
+        </article>
+      </section>
+    </main>
+
+    <main class="app-grid hidden" data-page="bookmark">
```
Comment (JelenadlS) [inline index.html:81]: I am not sure about this, but I think we should use only one main in a body and then you could separate the "pages" by using either divs or sections 😊

File: index.html [hunk 2]
```diff
@@ -9,111 +9,271 @@
...
+      </section>
+    </main>
 
-    <main class="app-grid__scrollarea">
-      <article class="card">
-        <h2 class="card__h2">Question:</h2>
-        <button
-          class="card__bookmark card__bookmark--style card__bookmark--clicked"
-          data-js="bookmark-clicked"
+    <main class="app-grid hidden" data-page="create">
```
Comment (JelenadlS) [inline index.html:150]: also here another main

File: index.html [hunk 3]
```diff
@@ -9,111 +9,271 @@
...
+            required
+            class="survey__tags-field"
+            data-js="surveyTags"
+          />
+          <button type="submit" class="survey__button">Submit</button>
+        </form>
+      </section>
+    </main>
+
+    <main class="app-grid hidden" data-page="profil">
```
Comment (JelenadlS) [inline index.html:202]: and another 😊

File: index.html [hunk 4]
```diff
@@ -9,111 +9,271 @@
...
+              maxlength="200"
+              required
+              class="survey__textarea"
+              data-js="surveyTextarea"
+            ></textarea>
+          </label>
+          <label for="tags-field" class="survey-label" data-form="form"
+            >Add your Tags:
+            <output class="survey__output--tags" data-js="output-tags"
+              >100</output
```
Comment (JelenadlS) [inline index.html:187]: I think it should be 200, as your maxlength and also down in the js file you wrote 200 😊

File: js/navbar.js [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+      });
+      const buttonAttribute = event.currentTarget.getAttribute('data-nav');
+      const currentPage = document.querySelector(
+        `[data-page="${buttonAttribute}"]`
+      );
+      currentPage.classList.remove('hidden');
+    });
+  });
+
+  const buttonHome = document.querySelector('[data-nav="home"]');
```
Comment (JelenadlS) [inline js/navbar.js:18]: Good job with including the short version - so you can take those lines below  out right? 😊
or did i oversee something?😊

**Review by JelenadlS — CHANGES_REQUESTED** [review]
It looks very good, great job and super easy to read through due to your well structured code 🤩 
I wrote you something in regarding the main in the index.html - i am not pretty sure about this to be honest as well as at the navigation.js - however, i might be wrong 😊

File: index.html (hunk 1 above)
Comment (celinevo) [reply index.html:81]: I thought that at first as well! But in the session last friday Jerry said we can use multible mains in out HTML. :)

File: index.html (hunk 4 above)
Comment (celinevo) [reply index.html:187]: 200 is for my textareas. The 100 charaktars are for my tags. I didn´t want there to be as long of a tag as there is a question or answer. I put those in a seperate js file.

File: js/navbar.js (hunk 5 above)
Comment (celinevo) [reply js/navbar.js:18]: Thank you! I am still surprised, that it worked without a problem. But I didn´t figure out yet, how to change the buttons to active in the same manner. So I used my old version for the buttons just to make it work. In my next branch I am going to work on that problem. Thank you very much for your nice review! 😸

**Pull request merged**